### DISCUSSION TITLE
SteamOS build

### DISCUSSION THREAD

**Discussion by Blood-PawWerewolf** [OpenRCT2/OpenRCT2#22956] [Feature requests] [answered]
there should be a separate build for SteamOS due to the GLIBC libraries being upgraded for OpenRCT2, yet are now no longer compatible (GLIBC on SteamOS is 2.7, the OpenRCT2 AppPackage now requires GLIBC 2.8). having to use the windows version via Proton works, but it's not recommended.

**Comment by TreeOfSelf** [comment]
Not only a separate SteamOS build, this upgrade broke it Ubuntu 22 users, and broke it for the latest stable release of Debian.

**Comment by Gymnasiast** [comment, marked as answer]
As of today, develop builds of the AppImage work on Ubuntu 22.04 and Debian Bookworm again. Probably also on SteamOS, but I haven’t specifically tested it. This change will also filter through the v0.4.16 release (planned for early November).
